scripts/commands/playlist/generate.ts

```diff
@@ -104,7 +104,15 @@ async function loadStreams({
   let streams = await parser.parse(files)
 
   streams = streams
-    .orderBy([(stream: Stream) => stream.channel, (stream: Stream) => stream.url], ['asc', 'asc'])
+    .orderBy(
+      [
+        (stream: Stream) => stream.channel,
+        (stream: Stream) => stream.timeshift,
+        (stream: Stream) => parseInt(stream.quality.replace('p', '')),
+        (stream: Stream) => stream.label
```

Comment:
Maybe you can sort by host as well? So that a YouTube link is only used if there's no other option. Thanks

Reply:
What's wrong with the YouTube link?

Reply:
It requires an extractor like https://ythlsgo.onrender.com/channel/ which goes down

Reply:
Well if the link doesn't work it should just be removed from the playlist

Reply:
Wouldn't it reduce the load on the extractor and thus it won't go down as much?

Reply:
I don't know, it's better to ask the author of this service about it.

Reply:
Broken links removed in https://github.com/iptv-org/iptv/pull/14940